public/app/plugins/datasource/alertmanager/ConfigEditor.tsx

```diff
@@ -36,7 +42,7 @@ export const ConfigEditor = (props: Props) => {
             <Select
               width={40}
               options={IMPL_OPTIONS}
-              value={options.jsonData.implementation || AlertManagerImplementation.cortex}
+              value={options.jsonData.implementation || AlertManagerImplementation.mimir}
```

Comment:
I changed the default here to Mimir, I think this should be fine since the endpoints are identical.